src/config/config.c

```diff
@@ -1074,6 +1074,12 @@ void initConfig(struct config *conf)
 	conf->misc.addr2line.f = FLAG_ADVANCED_SETTING;
 	conf->misc.addr2line.d.b = true;
 
+	conf->misc.etc_dnsmasq_d.k = "misc.etc_dnsmasq_d";
+	conf->misc.etc_dnsmasq_d.h = "Should FTL load additional dnsmasq configuration files from /etc/dnsmasq.d/?";
+	conf->misc.etc_dnsmasq_d.t = CONF_BOOL;
+	conf->misc.etc_dnsmasq_d.f = FLAG_ADVANCED_SETTING;
```

Comment:
This needs the restartflag as well.
```suggestion
	conf->misc.etc_dnsmasq_d.f = FLAG_RESTART_FTL | FLAG_ADVANCED_SETTING;
```